### PR TITLE
fix: warning for brew

### PR DESCRIPTION
**Pull request by castrojo**

### CODE REVIEW

File: just/custom.just [hunk 1]
```diff
@@ -12,8 +12,20 @@ bluefin-cli:
...
+    source /usr/lib/ujust/ujust.sh
     if [[ ! -f "/var/home/linuxbrew/.linuxbrew/bin" || ! -x "/var/home/linuxbrew/.linuxbrew/bin/brew" ]]; then
+        echo "${b}Brew Installation${n}"
+        echo "Please ${b}IGNORE${n} everything the installer tells you to do"
+        echo "We have already done it for you! You just need to close and re-open the terminal after installation"
+        echo "Do you understand?"
+        echo "Please type in \"Yes I understand\" and press enter"
+        read ACCEPT
+        if [ "$ACCEPT" == "Yes I understand" ]; then
+           if [[ ! -f "/var/home/linuxbrew/.linuxbrew/bin" || ! -x "/var/home/linuxbrew/.linuxbrew/bin/brew" ]]; then
```
Comment (HikariKnight) [inline just/custom.just]: remove this line

File: just/custom.just [hunk 2]
```diff
@@ -12,8 +12,20 @@ bluefin-cli:
...
         /usr/bin/bash -c '/usr/libexec/brew-install'
```
Comment (HikariKnight) [inline just/custom.just]: this must be indented

**Review by HikariKnight — APPROVED** [review]

File: just/custom.just [hunk 3]
```diff
@@ -12,8 +12,19 @@ bluefin-cli:
...
 brew:
     #!/usr/bin/env bash
+    source /usr/lib/ujust/ujust.sh
     if [[ ! -f "/var/home/linuxbrew/.linuxbrew/bin" || ! -x "/var/home/linuxbrew/.linuxbrew/bin/brew" ]]; then
-        /usr/bin/bash -c '/usr/libexec/brew-install'
+        echo "${b}Brew Installation${n}"
+        echo "Please ${b}IGNORE${n} everything the installer tells you to do"
+        echo "We have already done it for you! You just need to close and re-open the terminal after installation"
+        echo "Do you understand?"
+        echo "Please type in \"Yes I understand\" and press enter"
```
Comment (HikariKnight) [inline just/custom.just]: you forgot to change Yes I understand here

**Review by HikariKnight — COMMENTED** [review]
lgtm now

**Review by HikariKnight — APPROVED** [review]

**Comment by bketelsen** [comment]
LOL

**Pull request merged**